app/views/users/show.html.erb

```diff
@@ -1 +1,36 @@
-<h1>Here is a given user</h1>
\ No newline at end of file
+<h1>Here is a given user</h1>
+
+  <div class="to_all_container">
+    <img class="user_photo" src="<%= @user.photo %>" alt="user photo">
+    <div>
+      <h2>Name: <%= @user.name %></h2>
+      <h4>Number of posts: <%= @user.post_counter %></h4>
+    </div>
+ </div>
+
+ <div class="bio">
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+
+
+
+
+<div class="posts">
+  <% @posts.each do |post| %>
```

Comment:
- [ ] The posts that are shown on this page should be the recent posts using the method you have in the user model. Kindly change to that.